{
  try {
    return expand(operations,list,null,true);
  }
 catch (  FactoryException exception) {
    throw new AssertionError(exception);
  }
}
